syle.css

```
@@ -37,4 +37,12 @@ body {
 .maintitle {
     font-size: 100px;
     font-weight: bolder;
+}
+.foot{
+    background-color: blueviolet;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    font-weight: bolder;
+    font-size: 20px;
```

```suggestion
    font-weight: bolder;
    font-size: 20px;
```
these are making h1 and b tags useless, you should move them to `THE STORY`s own class